test/unit/specs/components/govern/LiProposal.spec.js

```diff
@@ -0,0 +1,58 @@
+import { shallow } from "@vue/test-utils"
+import LiProposal from "renderer/components/govern/LiProposal"
+
+describe("LiProposal", () => {
+  let wrapper
+  let propsData = {
```

Comment:
It's better to use `const` by default and `let` only when necessary because `const` preserves [referential transparency](https://en.wikipedia.org/wiki/Referential_transparency).

Reply:
Makes sense, `propsData` is entirely static.